### DISCUSSION TITLE
Support for UUID?

### DISCUSSION THREAD

**Discussion by pgregorio-signalytic** [xerial/sqlite-jdbc#858] [Q&A] [answered]
Hi, I'm looking for a SQLite JDBC driver to use on my project and I need it to support [UUID()](https://sqlite.org/src/file/ext/misc/uuid.c). Can someone quickly confirm if this JDBC driver does? Is it enabled by default or do I need additional configuration to enable it?

**Comment by gotson** [comment, marked as answer]
This extension is not built in the native library the driver uses. You can probably compile the extension yourself and load it dynamically.
